Resources/Prototypes/_Mono/Shipyard/Expedition/picses.yml

```
@@ -0,0 +1,40 @@
+  parent: BaseVessel
+  name: TSK Picses
+  description: A compact but capable expedition ship, with durable hull plating too.
+  price: 85250 # plastitanium tax...
+  category: Medium
+  group: Expedition
+  shuttlePath: /Maps/_Mono/Shuttles/Expedition/picses.yml
+  guidebookPage: null
+  class:
+  - Expedition
+  engine:
+  - AME
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)